### ISSUE TITLE
lower results than expected

### ISSUE THREAD

**Issue by lcysonya** [guochengqian/PointNeXt#14]
Thanks for the great work！

I use 4 gpus to run script/main_segmentation.sh cfgs/s3dis/pointnext-xl.yaml. Only got val_oa 89.21, val_macc 76.35, val_miou 69.08 (Best ckpt @E61) which lower than the results you showed at homepage.

Is it a normal results,  or is there some expriment settings I haven't noticed that effect the results.

**Comment by guochengqian** [comment]
how about training using only 1 GPU?

**Comment by lcysonya** [comment]
Thanks a lot. It works. 
I thought bsize wouldn't have much effect

**Comment by haibo-qiu** [comment]
Hi there~
Thanks for your great open-source work!

Due to the limited memory of my GPU, I use **4 GPUS with batch size 2** under SyncBN to simulate 1 GPU with batch size 8.
Then I get the results `ckpts @ 52 epoch( {'best_val': 69.06454467773438} )`. I test this model on the test set and obtain `test_oa , test_macc, test_miou: 90.37 76.72 70.22`, which matches the original results [70.5±0.3](https://github.com/guochengqian/PointNeXt/blob/master/README.md#s3dis-area-5-segmentation) but inferior to your 71.1

Is that performance normal? or do I miss something important?

Note that I get the errors `CrossEntropy, unexpected param` *`label_smoothing`* when first running the code. After I check the original paper and the loss implementation, I think this criterion should change to `SmoothCrossEntropy`. The above results are obtained after making this change. 

By the way, @lcysonya thought the batch size wouldn't have much effect. But in my situation, when I use **4 GPUS with batch size 4** for training (largest size by my GPU), I finally get the val results `ckpts @ 93 epoch( {'best_val': 68.37361907958984} )` and testing performance `test_oa , test_macc, test_miou: 90.53 75.90 69.78` which is clearly lower than batch size 2.

I might get something unexpected, but I indeed do not change other settings. I would much appreciate it if you (@lcysonya)  can tell me how you solved your problem.

**Comment by guochengqian** [comment]
Batch size and multi gpu tranining influence a decent margin (eg. at least 2 miou at S3DIS) in point cloud experiments.  If you are familiar with the literature, you will find point cloud domain never uses larger batch size training.  
if you trained with the exact same configuration as us, you will get the results promised (make sure it is testing result not validation).

**Comment by guochengqian** [comment]
@haibo-qiu i think it is normal. Just run three times and report the mean. One experiment cannot show much.

**Comment by guochengqian** [comment]
71.1 is the best we get. Please refer to the mean results reported in paper

**Comment by zen-d** [comment]
@haibo-qiu I wonder how to conduct multi-GPU segmentation training with the current code since as mentioned by @guochengqian, there exists an issue #8 that is unsolved yet.

**Comment by haibo-qiu** [comment]
@guochengqian Thanks for your kind reply.

**Comment by haibo-qiu** [comment]
@zen-d Actually [this comment](https://github.com/guochengqian/PointNeXt/issues/8#issuecomment-1158360354) by Wang solved that issue, i.e., simply adding a comma to change `mp.spawn(main, nprocs=cfg.world_size, args=(cfg))` to `mp.spawn(main, nprocs=cfg.world_size, args=(cfg,))`.
